lib/editor/editor.dart

```diff
@@ -106,7 +106,7 @@ class EditorState extends State<Editor> {
       }
     }
 
-    if (!widget.options.hasRegion) {
+    if (widget.options.regionOptions == null) {
```

Comment:
Now that I think about this, it can be the else part of the above if-block or vice-versa